tests/app/questionnaire/test_questionnaire_schema.py

```diff
@@ -782,7 +782,7 @@ def test_progress_block_dependencies(
     schema = progress_block_dependencies_schema
 
     assert {
-        "section-1": {"calculated-summary-block": ["section-2", "section-3"]}
+        "section-1": {"calculated-summary-block": {"section-2", "section-3"}}
```

Comment:
Appreciate this is the case in lots of places, so more a question, but does it matter that the types in the tests are inconsistent with the specific type, so set vs ordered set? or is it generally fine if they are of the same generic type? this works in any case so not against it

Reply:
We should try to make the test as close as possible to the real thing. If we type hinted test files, that would raise a LOT of issues, it would pick up this like this. Happy to revisit as a whole.

Reply:
Updated this particular test 👍